.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
This file should *not* be included.  In 9448fe2fc7f282afed471b4afe67e42c86b8f859 I added `.eggs` to gitignore, but in the meantime this should not be checked in in this PR.